docs/registry/access-mgmt-registry/README.adoc

```diff
@@ -137,11 +139,39 @@ You can edit or remove the user roles assigned in your {registry} instances that
 .Procedure
 . In the web console, go to *{registry}* > *{registry} Instances* and click the name of the {registry} instance that you want to remove a user role for.
 . Click the *Access* tab to view the accounts and roles assigned for this instance.
-. Select the options menu (three vertical dots) next to the assigned *Role* name: 
-.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.  
+. Select the options menu (three vertical dots) next to the assigned *Role* name:
+.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.
 .. To remove the currently assigned role, click *Remove* and confirm in the dialog.
 
+[id="proc-user-account-access_{context}"]
+== Using RBAC to allow users to view other user accounts
 
+[role="_abstract"]
+As an organization administrator, you can use Role-Based Access Control (RBAC) to allow users to view other users in a group.
+
+You set up access by assigning a predefined role called `User Access principal viewer` to a user group.
+By assigning the role, users within the group are able to do the following:
+
+* View and select other users when changing owners and managing access to Service Registry instances in the web console
+* Specify user names when managing the roles of Service Registry instances using the `rhoas service-registry role` commands of the `rhoas` CLI for {product-long-registry}
```

Comment:
I wonder if we should just keep the specifics of what can be done in the CLI more vague? Otherwise I can see this either:

* having to be updated. regularly as more CLI use cases are added
* falling behind and leading the reader to believe that these are the only two things that are enabled with this role when we could have added more.